docs-src/cloud/operating-envelope-latency.md

```diff
@@ -0,0 +1,16 @@
+---
+id: operating-envelope-latency
+title: What kind of latency can I expect from Temporal Cloud?
+sidebar_label: Latency
+description: Our latency SLO is 200ms per region for p99.
+tags:
+  - temporal-cloud
+  - operations
+  - explanation
+---
+
+Our latency SLO is 200ms per region for p99.
+Latency is greatly influenced by the actual throughput of a single [Workflow Execution](/concepts/what-is-a-workflow-execution).
+Concurrent operations on the same Workflow Execution could lead to increased latency.
+
+Typical latency for starting and signaling Workflow Executions is less than 100ms.
```

Comment:
```suggestion
In June 2023, latency measurements over a week period for starting and signaling Workflow Executions were:

- `StartWorkflowExecution`: 90ms p90, 125ms p99
- `SignalWorkflowExecution`: 53ms p90, 95ms p99
- `SignalWithStartWorkflowExecution`: 87ms p90, 116ms p99

We work on improving latency, so these numbers will decrease over time.

Concurrent operations on the same Workflow Execution could lead to increased latency.
```

throughput of WE doesn't make sense to me. if talking about latency of completion of a WE, that's a different topic.